backend/src/Core/FurnitureShop.Core.Services/CQRS/Mobile/Products/ProductByIdQH.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using FurnitureShop.Core.Contracts.Mobile.Products;
+using FurnitureShop.Core.Domain;
+using FurnitureShop.Core.Services.DataAccess;
+using Microsoft.EntityFrameworkCore;
+
+namespace FurnitureShop.Core.Services.CQRS.Mobile.Products
+{
+    public class ProductByIdQH : IQueryHandler<ProductById, ProductDTO?>
+    {
+        private readonly CoreDbContext dbContext;
+
+        public ProductByIdQH(CoreDbContext dbContext)
+        {
+            this.dbContext = dbContext;
+        }
+
+        public async Task<ProductDTO?> ExecuteAsync(CoreContext context, ProductById query)
+        {
+            return await dbContext.Products
```

Comment:
Is it some dummy data? Can u just put it into seed for the empty base and try to write it instead of changing it later?

Reply:
a zostało mi gdzies to jeszcze bedzie normalnie luz